vcr/vcr.go

```diff
@@ -257,18 +257,8 @@ func (c *vcr) Configure(config core.ServerConfig) error {
 		// meaning while the issuer allocated an HTTP connection the wallet will try to allocate one as well.
 		// This moved back to 1 http.Client when the credential is requested asynchronously.
 		// Should be fixed as part of https://github.com/nuts-foundation/nuts-node/issues/2039
-		issuerTransport := http.DefaultTransport.(*http.Transport).Clone()
-		issuerTransport.TLSClientConfig = tlsConfig
-		c.issuerHttpClient = core.NewStrictHTTPClient(config.Strictmode, &http.Client{
-			Timeout:   c.config.OpenID4VCI.Timeout,
-			Transport: issuerTransport,
-		})
-		walletTransport := http.DefaultTransport.(*http.Transport).Clone()
-		walletTransport.TLSClientConfig = tlsConfig
-		c.walletHttpClient = core.NewStrictHTTPClient(config.Strictmode, &http.Client{
-			Timeout:   c.config.OpenID4VCI.Timeout,
-			Transport: walletTransport,
-		})
+		c.issuerHttpClient = core.NewStrictHTTPClient(config.Strictmode, c.config.OpenID4VCI.Timeout, tlsConfig)
```

Comment:
Now they use the same connection pool? (see comment above)

Reply:
abstracted to core.NewStrictHTTPClient with notes.